### PR TITLE
Links and buttons updates

### PR DESCRIPTION
**Pull request by tritland**
Updates to How To Test Links and Buttons:

- Used a div button instead of negative tabindex as non-focusable example
- Updated disabled button example to look disabled but still be actionable
- updated the last ("current") breadcrumb link to be bold

### CODE REVIEW

**Comment by joe-watkins** [comment]
@tritland Looks good to me but I think there are changes in this PR that are the same/similar to https://github.com/tmobile/magentaA11y/pull/75/files/8c272a7530b6e501372af9b8caef291cd34803ac#r1336191121 ? Don't want ya to run into any conflicts.

File: how-to/form.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ Select fields allow users to select one or more options from an expandable list
 
 <div class="how-to-test-checklist-item">
   <h3>✓ Ensure there is proper error handling for required fields</h3>
-  <p><strong>Note:</strong> When error text is not programmatically associated by use of <code>aria-describedby</code> they may not be discovered by screen reader users.</p>
+  <p><strong>Note:</strong> Error messages should be announced for screen reader users when the form recives focus. Forms should also notify screen reader users if invalid content has been entered in a field.</p>
```
Comment (joe-watkins) [inline how-to/form.md:300]: Similar changes in #75 look out for conflicts. https://github.com/tmobile/magentaA11y/pull/75/files/8c272a7530b6e501372af9b8caef291cd34803ac#r1336191121

**Review by joe-watkins — COMMENTED** [review]
Similar code changes to the

File: how-to/form.md (hunk 1 above)
Comment (tritland) [reply how-to/form.md]: Cannot figure out how this file made it in here! Thanks for heads up

Comment (joe-watkins) [reply how-to/form.md]: @tritland  You may have checked out the branch you did that work in for this branch instead of `main` ?

**Review by jzoo — APPROVED** [review]
looks good to me

**Review by joe-watkins — COMMENTED** [review]
Looks good!

**Pull request merged**